R/arc-select.R

```diff
@@ -224,26 +233,27 @@ collect_layer <- function(
   # count the number of features in a query
   n_feats <- count_results(req, query_params)
 
-  if (is.null(n_feats)) {
-    cli::cli_abort(c(
-      "Cannot determine the number of features in request.",
-      "i" = "did you set custom parameters via {.arg ...}?"
```

Comment:
this message is really important because dots is used to pass in extra arguments that might not be exposed via function arguments _or_ it might have captured mistyped names in the arguments. I would like to keep some variety of this